### PR TITLE
Updates the code sample for aiplatform(unified) Vizier

### PR DESCRIPTION
**Pull request by halio-g**
Tested by running commands: 

python3 -m tensorflow_docs.tools.nbfmt ai_platform_unified_vizier_multi_objective.ipynb
pre-commit run --files ai_platform_unified_vizier_multi_objective.ipynb

### CODE REVIEW

File: ai-platform-unified/notebooks/official/gapic/ai_platform_unified_vizier_multi_objective.ipynb [hunk 1]
```diff
@@ -0,0 +1,513 @@
+{
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/gapic/ai_platform_unified_vizier_multi_objective.ipynb:1]: <p>This was updated in the template so you probably didn't see it yet.</p><p> </p><p>Change to:</p><p> </p><pre class="ql-syntax" spellcheck="false">elif not os.getenv("IS_TESTING"):
    %env GOOGLE_APPLICATION_CREDENTIALS your_path_to_credentials.json
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by ivanmkc — APPROVED** [review]
Nice job! The code looks fine so perhaps get a quick look from your tech writer partner. Thanks!

**Comment by halio-g** [comment]
Thanks for the review. I'll merge the pr.

**Pull request merged**